src/test/java/net/sf/jabref/logic/formatter/bibtexfields/ClearFormatterTest.java

```diff
@@ -1,18 +1,8 @@
 package net.sf.jabref.logic.formatter.bibtexfields;
 
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
+/**
+ * Tests in addition to the general tests from {@link net.sf.jabref.logic.formatter.FormatterTest}
+ */
 public class ClearFormatterTest {
 
-    @Test
-    public void formatReturnsEmptyForEmptyString() throws Exception {
-        assertEquals("", new ClearFormatter().format(""));
-    }
-
-    @Test
-    public void formatReturnsEmptyForSomeString() throws Exception {
```

Comment:
Fixed (via changing the commit "Formatters tests: Remove methods already covered by FormatterTest" from 9de60b5b8cf04 into 1262c2551e266.